### DISCUSSION TITLE
Download onto s3 pvc is slow

### DISCUSSION THREAD

**Discussion by mahi402** [datashim-io/datashim#245] [General]
I have installed the dataset onto eks cluster, when i download the onto pod my mmounting the pvc, the speed is slow
like 10 mb is causing 3 to 5 mins..
when i download to other folder in the pod, its fast

any solution will be of great help

**Comment by srikumar003** [comment]
@mahi402 is the bucket in the same region as the cluster hosting the pod ? What would help is the logs of the `csi-s3`  daemons (please remove any passwords or tokens or bucket names)

**Reply by mahi402** [reply]
yes, the bucket is in same region, please find logs

Defaulted container "driver-registrar" out of: driver-registrar, csi-s3
I0330 18:04:33.986814       1 main.go:164] Version: v2.3.0
I0330 18:04:33.986839       1 main.go:165] Running node-driver-registrar in mode=registration
I0330 18:04:33.987521       1 main.go:189] Attempting to open a gRPC connection with: "/csi/csi.sock"
I0330 18:04:33.987541       1 connection.go:154] Connecting to unix:///csi/csi.sock
I0330 18:04:35.979600       1 main.go:196] Calling CSI driver to discover driver name
I0330 18:04:35.979628       1 connection.go:183] GRPC call: /csi.v1.Identity/GetPluginInfo
I0330 18:04:35.979632       1 connection.go:184] GRPC request: {}
I0330 18:04:35.984351       1 connection.go:186] GRPC response: {"name":"ch.ctrox.csi.s3-driver","vendor_version":"v1.1.1"}
I0330 18:04:35.984410       1 connection.go:187] GRPC error: <nil>
I0330 18:04:35.984416       1 main.go:206] CSI driver name: "ch.ctrox.csi.s3-driver"
I0330 18:04:35.984434       1 node_register.go:52] Starting Registration Server at: /registration/ch.ctrox.csi.s3-driver-reg.sock
I0330 18:04:35.984594       1 node_register.go:61] Registration Server started at: /registration/ch.ctrox.csi.s3-driver-reg.sock
I0330 18:04:35.984645       1 node_register.go:91] Skipping healthz server because HTTP endpoint is set to: ""
I0330 18:04:36.393034       1 main.go:100] Received GetInfo call: &InfoRequest{}
I0330 18:04:36.393226       1 main.go:107] "Kubelet registration probe created" path="/var/lib/kubelet/plugins/csi-s3/registration"
I0330 18:04:36.427337       1 main.go:118] Received NotifyRegistrationStatus call: &RegistrationStatus{PluginRegistered:true,Error:,}
